utils/github/query.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+
+import requests
+
+
+class Query:
+    def __init__(self, token, max_page_size=100, pull_request_page_size=5):
+        self._token = token
+        self._max_page_size = max_page_size
+        self._pull_request_page_size = pull_request_page_size
+
+    ''' Get all pull-requests associated with at least one commit in range (until, default head],
+        where "default head" is a head of default repository branch.
+    '''
+    def get_pull_requests(self, until):
+        pull_requests = []
+        query = Query._FIRST.format(max_page_size=self._max_page_size, pull_request_page_size=self._pull_request_page_size)
```

Comment:
Query template names are unobvious. What is `_FIRST` or `_NEXT`?

Reply:
They names stand for the first query, and for the all next queries in a row: the difference is in the pagination param `cursor` (`history(… after: cursor)`). In the first query we don't know even cursor format, so we can't specify this param, but we know and need it to request all data further on. The better solution is to form both queries using some Python GraphQL compositor - it's a todo for the future. I can rename strings, if you suggest better names.